src/CSharp/EasyMicroservices.SupportsMicroservice.WebApi/Controllers/DepartmentController.cs

```diff
@@ -3,14 +3,21 @@
 using EasyMicroservices.SupportsMicroservice.Contracts.Common;
 using EasyMicroservices.SupportsMicroservice.Contracts.Requests;
 using EasyMicroservices.SupportsMicroservice.Database.Entities;
+using EasyMicroservices.Cores.AspEntityFrameworkCoreApi.Interfaces;
+
+
 
 namespace EasyMicroservices.SupportsMicroservice.WebApi.Controllers
 {
     public class DepartmentController : SimpleQueryServiceController<DepartmentEntity, CreateDepartmentRequestContract, UpdateDepartmentRequestContract, DepartmentContract, long>
     {
-        public DepartmentController(IContractLogic<DepartmentEntity, CreateDepartmentRequestContract, UpdateDepartmentRequestContract, DepartmentContract, long> contractLogic) : base(contractLogic)
-        {
+        private readonly IContractLogic<DepartmentEntity, CreateDepartmentRequestContract, UpdateDepartmentRequestContract, DepartmentContract, long> _contractlogic;
```

Comment:
You don't need to use contractlogic property. use unitofwork inside of your methods